mmdet/models/dense_heads/rpn_test_mixin.py

```diff
@@ -13,20 +10,20 @@ class RPNTestMixin(object):
 
```

Comment:
It may be more reasonable to rename it to `RPNHeadTestMixin`.

Reply:
Similar to ``BboxTestMixin`` and ``MaskTestMixin``, I think ``RPNTestMixin`` is reasonable.